lib/modules/datasource/docker/google.ts

```diff
@@ -0,0 +1,29 @@
+import { GoogleAuth } from "google-auth-library"
+import { logger } from '../../../logger';
+import { regEx } from '../../../util/regex';
+import { addSecretForSanitizing } from '../../../util/sanitize';
+
+export const googleRegex = regEx(/([a-z-]+\.)?(gcr.io|pkg.dev)/);
```

Comment:
`pkg.dev`: are you sure it can also be a docker registry? I only know `gcr.io` as registry

Reply:
Thx for reviewing! Yes. `pkg.dev` is the domain for [Artifact Registry](https://cloud.google.com/artifact-registry). URLs for Artifact Registry generally look like this:
`<location>-docker.pkg.dev`
Location can be a single region (`us-central1`) or multi-region (`europe`).

Reply:
then make the regex more strict to only match the docker part